pelican/settings.py

```diff
@@ -101,10 +101,12 @@ def load_source(name, path):
     'PELICAN_CLASS': 'pelican.Pelican',
     'DEFAULT_DATE_FORMAT': '%a %d %B %Y',
     'DATE_FORMATS': {},
-    'MD_EXTENSIONS': {
-        'markdown.extensions.codehilite': {'css_class': 'highlight'},
-        'markdown.extensions.extra': {},
-        'markdown.extensions.meta': {},
+    'MARKDOWN': {
+        'extension_configs': {
+            'markdown.extensions.codehilite': {'css_class': 'highlight'},
+            'markdown.extensions.extra': {},
+            'markdown.extensions.meta': {},
+        },
```

Comment:
Sooner or later that should happens (from Python-Markdown side of from Pelican one). Sooner is better. Only one note: it is breaking change and should be mentioned in the release notes.